src/azul/http.py

```diff
@@ -8,6 +8,9 @@
 )
 
 import certifi
+from chalice import (
+    TooManyRequestsError,
+)
```

Comment:
We can't drag Chalice into this. We need to use a custom exception and translate that to the Chalice exception in the controller layer. I can explain why in PL if needed. Model this after `LimitedTimeOutException`.